lib/api/core/models/repositories/roleRepository.js

```diff
@@ -173,7 +173,7 @@ module.exports = function (kuzzle) {
    */
   RoleRepository.prototype.validateAndSaveRole = function (role) {
     var context = {
-      connection: {type: 'websocket'},
+      connection: {type: 'internal'},
```

Comment:
I was afraid of potential side-effects, and too lazy to investigate further :stuck_out_tongue: 

It seems this function is only called by the security controller, so I'll add the real context to this one, and to other similar functions used by profiles and users management.